Source/DafnyDriver/Commands/CommandRegistry.cs

```diff
@@ -188,7 +188,14 @@ class WritersConsole : IConsole {
         }
 
         options.OptionArguments[option] = value;
-        dafnyOptions.ApplyBinding(option);
+        try {
+          dafnyOptions.ApplyBinding(option);
+        } catch (Exception e) {
+          failedToProcessFile = true;
```

Comment:
Could you rename `failedToProcessFile` to `errorOccurred` ?

Reply:
OK ( though the name was chosen before this PR)